app/src/main/java/com/hedvig/app/util/apollo/PaymentMethodsApiResponseAdapter.kt

```diff
@@ -6,6 +6,8 @@ import com.apollographql.apollo3.api.CustomTypeValue
 import com.hedvig.app.util.toJsonObject
 import org.json.JSONObject
 
+// TODO: Not sure why we're doing the LinkedHashMap check and how to migrate this to apollo3 Adapter. We can use the
```

Comment:
Good question - not sure. But given that Apollo 3 comes with different type guarantees, I guess this should be just fine to convert to the new API.

Reply:
Alright, I'll keep this in mind and try with and without migrating this and see how the app functions in cases where it parses this.
Also as I see that Hugo was the one who touched this part last I will consult him too to see if there was something we need to be wary of in this process.